tuf/repository_tool.py

```diff
@@ -74,8 +74,14 @@
 
 from securesystemslib.interface import (
     generate_and_write_rsa_keypair,
+    generate_and_write_rsa_keypair_with_prompt,
```

Comment:
Do we actually use these new functions here, or are they just being imported w/o being used?

Reply:
The latter is correct. They are just added to the TUF namespace (see comments above), because a lot of docs suggest to do `from repository_tool import *` to access them.

I started this in https://github.com/theupdateframework/tuf/pull/919 (see 7306446118b89e3e395340e748a2717b03e0666e for details), in order to remove many oneliner function wrappers that served the same purpose but added cruft to the code base and duplicated the maintenance burden above all in regards to docstrings.

A better but also more invasive fix would be to remove these functions from the  tuf namespace altogether, and update the docs to directly use securesystemslib akin to what we now do in in-toto (see https://github.com/in-toto/in-toto/pull/402, https://github.com/in-toto/in-toto/pull/408 and https://in-toto-lukpueh.readthedocs.io/en/latest/api.html#key-utilities).